{"name": "test_open_about", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"af264618169df641b8b311d4a7eb1c9b\")>\n\n    @allure.description(\"test_open_about\")\n    def test_open_about(driver):\n        log = LoginPage(driver)\n        log.autorization()\n        mp = MainPage(driver)\n        mp.check()\n>       mp.selectMenuAbout()\n\ntests\\test_open_about.py:13: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npages\\mainPage.py:73: in selectMenuAbout\n    self.clickAboutPage()\npages\\mainPage.py:57: in clickAboutPage\n    self.getLinkAbout().click()\npages\\mainPage.py:38: in getLinkAbout\n    return WebDriverWait(self.driver, 30).until(EC.element_to_be_clickable((By.XPATH, self.linkAbout)))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"af264618169df641b8b311d4a7eb1c9b\")>\nmethod = <selenium.webdriver.support.expected_conditions.element_to_be_clickable object at 0x000002AAF27CFFA0>, message = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n..\\..\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:80: TimeoutException"}, "description": "test_open_about", "steps": [{"name": "autorization", "status": "passed", "start": 1680725509503, "stop": 1680725511696}, {"name": "check", "status": "passed", "start": 1680725511696, "stop": 1680725511802}], "attachments": [{"name": "stdout", "source": "aff43ba3-22ae-4502-bb5b-06798ca9876f-attachment.txt", "type": "text/plain"}], "start": 1680725509503, "stop": 1680725548579, "uuid": "4d18eb40-c3e2-4167-ad60-ae70f17e9e26", "historyId": "33ce6c50b62156342adae983e3e1c18c", "testCaseId": "045d8929b692b41884b3edb777820813", "fullName": "tests.test_open_about#test_open_about", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_open_about"}, {"name": "host", "value": "LAPTOP-70OVIOCN"}, {"name": "thread", "value": "17836-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_open_about"}]}